src/ghost/ghost-post.service.ts

```diff
@@ -95,4 +101,23 @@ export class GhostPostService {
             }
         }
     }
+
+    async deleteGhostPost(
+        account: Account,
+        uuid: string,
+    ): Promise<Result<boolean, DeletePostError>> {
+        const apId = account.getApIdForPost({
+            uuid,
+            type: PostType.Article,
+        });
+
+        const deleteResult = await this.postService.deleteByApId(apId, account);
+        if (isError(deleteResult)) {
+            this.logger.error(
+                'Failed to delete post with apId: {apId}, error: {error}',
+                { apId, error: getError(deleteResult) },
+            );
+        }
```

Comment:
now you are returning the result back to the controller, should the logging happen there instead of here?